src/encoded/static/components/facets/audit_processed_data.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import url from 'url';
+import QueryString from '../../libs/query_string';
+import BooleanSwitch from '../../libs/ui/boolean-switch';
+import { SpecialFacetRegistry } from './registry';
+
+
+/**
+ * Special facet to display a switch to select between having the search result query string
+ * contain: "audit.WARNING.category!=lacking processed data" and without that term at all, with the
+ * boolean switch component showing the "on" state with the former and the "off" state with the
+ * latter.
+ */
```

Comment:
@hitz Do you have an opinion on this? This seemed OK to me since you have the full control at the bottom of the facets, and I’m not sure if a different color would mean anything to the user. So right now if you have no “audit.WARNING.category” or “audit.WARNING.category=lacking+processed+data” the switch appears off. If you have “ audit.WARNING.category!=lacking+processed+data” the switch appears on.